apps/native-component-list/package.json

```diff
@@ -48,7 +48,7 @@
   },
   "dependencies": {
     "@expo/react-native-action-sheet": "^3.8.0",
-    "@react-native-async-storage/async-storage": "~1.14.1",
+    "@react-native-async-storage/async-storage": "^1.15.0",
```

Comment:
Are you sure we want to use the whole major semver range here, or should we pin to an exact version like we do for the deps below?

Reply:
i was undecided about this... but we've got away with this for `@react-native-community/async-storage` so far. maybe i'll change to `~` to be safe